modules/uoscore-uedhoc/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if (CONFIG_UOSCORE OR CONFIG_UEDHOC)
+
+  set(MODULE_SRC_DIR ${ZEPHYR_CURRENT_MODULE_DIR}/src)
+  set(MODULE_INCLUDE_DIR ${ZEPHYR_CURRENT_MODULE_DIR}/inc)
+
+  zephyr_include_directories(${MODULE_INCLUDE_DIR})
+
+# Common part used by both libraries
+
+  zephyr_library_named(uoscore_uedhoc_common)
+
+  file(GLOB
+       uoscore_uedhoc_common_sources
+       ${MODULE_SRC_DIR}/common/*.c
+  )
```

Comment:
please avoid globbing, it's discouraged in [CMake](https://cmake.org/cmake/help/latest/command/file.html#glob).
> **Note**: We do not recommend using GLOB to collect a list of source files from your source tree. 

It's fragile, plus the fact that if no files are found then CMake won't complain, as opposed to a list of files where CMake will print an error if the file doesn't exist.

Reply:
Thanks for pointing this out, I've switched to providing file list directly.